orchestration/dagster/hca_orchestration/resources/base.py

```diff
@@ -0,0 +1,165 @@
+import subprocess
```

Comment:
This file contains the resources we use in non-test modes. See https://docs.dagster.io/overview/modes-resources-presets/modes-resources for more details